pkgs/applications/window-managers/picom/default.nix

```diff
@@ -35,10 +35,10 @@ stdenv.mkDerivation rec {
     libxdg_basedir
   ];
 
-  NIX_CFLAGS_COMPILE = "-fno-strict-aliasing";
+  mesonBuildType = "release";
```

Comment:
I wonder why this does not default to release.

Reply:
I mean the option in nixpkgs. I think the default value is plain.

Reply:
FYI, diff in derivation size is quite small. I failed at copy and paste in OP :facepalm: .

Reply:
The binary size itself decreased from 396k->394k (but the derivation size is slightly larger thanks to the manpages probably).